core/src/main/java/org/infinispan/util/logging/Log.java

```diff
@@ -138,7 +138,7 @@
 //   @Message(value = "DistributionManager not started after waiting up to 5 minutes! Not rehashing!", id = 15)
 //   void distributionManagerNotStarted();
 
-   @LogMessage(level = WARN)
+   @LogMessage(level = ERROR)
```

Comment:
Why the escalation in level?

Reply:
The idea is that if an entry is not transferred properly, the node will see it as missing, so it's as if a regular operation failed on some of the owners. Initially I wanted to interrupt state transfer if there's any exception, but then I relented because we don't have an easy way to recover from that yet :)

Reply:
Okay, I kinda figured, just was more curious.